eodag/rest/utils.py

```diff
@@ -411,7 +420,11 @@ def search_products(product_type, arguments, stac_formatted=True):
         # We remove potential None values to use the default values of the search method
         criterias = dict((k, v) for k, v in criterias.items() if v is not None)
 
-        products, total = eodag_api.search(**criterias)
+        try:
```

Comment:
- `requests.RequestException` should not be raised by eodag, or this is because one has not been properly catched.
- exceptions are handled in `server.py`

Reply:
exception handling not necessary anymore because requests.RequestException has been removed in other class